src/dialog/dlgabout.cpp

```diff
@@ -11,24 +11,20 @@ DlgAbout::DlgAbout(QWidget* parent) : QDialog(parent), Ui::DlgAboutDlg() {
     setupUi(this);
 
     QString mixxxVersion = Version::version();
-    QString buildBranch = Version::developmentBranch();
-    QString buildRevision = Version::developmentRevision();
+    QString buildCommitDescription = Version::gitCommitDescription();
+    QString buildCommitDate = Version::gitCommitDate();
 
-    QStringList version;
-    version.append(mixxxVersion);
-
-    if (!buildBranch.isEmpty() || !buildRevision.isEmpty()) {
-        QStringList buildInfo;
-        buildInfo.append("build");
-        if (!buildBranch.isEmpty()) {
-            buildInfo.append(buildBranch);
-        }
-        if (!buildRevision.isEmpty()) {
-            buildInfo.append(QString("r%1").arg(buildRevision));
-        }
-        version.append(QString("(%1)").arg(buildInfo.join(" ")));
+    QString versionInfo = tr("Mixxx version:") + QChar('\n');
+    // The Git commit description string is long, so put it on its own line
+    if (!buildCommitDescription.isEmpty()) {
+        versionInfo.append(buildCommitDescription);
+    } else {
+        versionInfo.append(mixxxVersion);
+    }
+    if (!buildCommitDate.isEmpty()) {
+        versionInfo += QChar('\n') + tr("Git commit date:") + QChar('\n') + buildCommitDate;
```

Comment:
```suggestion
    QString versionInfo = tr("Mixxx version:");
    // The Git commit description string is long, so put it on its own line
    if (!buildCommitDescription.isEmpty()) {
        versionInfo += QStringLiteral("  ") + buildCommitDescription;
    } else {
        versionInfo += QStringLiteral("  ") + mixxxVersion;
    }
    if (!buildCommitDate.isEmpty()) {
        versionInfo += QChar('\n') + tr("Git commit date:") + QStringLiteral("  ") + buildCommitDate;
    }
```

Reply:
I disagree. It is easier to copy and paste the version string when it is on its own line.